### DISCUSSION TITLE
Source for foundation type

### DISCUSSION THREAD

**Discussion by sklepac** [HydrologicEngineeringCenter/NSI#19] [Q&A] [answered]
Is it possible for a public user to determine whether a foundation type in the NSI came from parcel data or from HAZUS data?

I do not need to know the source of the parcel data (as in the private field "p_fndtype"), but only whether the foundation type was randomly assigned from HAZUS data as described in the Technical Documentation or not.

Please let me know if you have any questions or would like further clarification on my question.

Thank you!

**Comment by Nick-Lutz** [comment, marked as answer]
Unfortunately, no. This bit of obfuscation is one of the means of protecting our input datasets.

We're looking for ways to convey uncertainty in future versions while still obeying licensing agreements. In the meantime, I suggest using surveys to guide your assessment of the data's accuracy in your study area and to quantify uncertainty around the attributes.

**Reply by sklepac** [reply]
Thanks for the quick response and concise explanation!
